person_capture/gui_app.py

```
@@ -7362,6 +7362,18 @@ def _truthy_env(name: str) -> bool:
         val = os.getenv(name, "")
         return str(val).strip().lower() in {"1", "true", "yes", "on"}
 
+    @staticmethod
+    def _float_env(name: str, default: float, *, min_value: Optional[float] = None, max_value: Optional[float] = None) -> float:
+        try:
+            value = float(str(os.getenv(name, "")).strip() or float(default))
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)